docs/content/en/docs/clustermgmt/networking/networking-and-security.md

```diff
@@ -126,6 +126,12 @@ For next steps on leveraging Network Policy, we encourage you to explore:
 
 ## Additional Cilium Features
 
-Many advanced features of Cilium are not yet enabled as part of EKS Anywhere, including:  Hubble observability, DNS-aware and HTTP-Aware Network Policy, Multi-cluster Routing, Transparent Encryption, and Advanced Load-balancing.
+Many advanced features of Cilium are not yet enabled as part of EKS Anywhere, including:  
```

Comment:
Is this the list of Open Source features that are not enabled or Enterprise features?

Reply:
@raykrueger The default eks-a image has a very basic set of features as illustrated on that page. That's right these are EE features. The way we are going with this is a comparison amongst what is there in the embedded version of EKS-A for Cilium vs. what is there in Isovalent Enterprise for Cilium.

@danwent

Reply:
This conflicts with #6060